Run project

After installing all dependencies on mac I'm running main.py and getting this on my console:

ld: library not found for -lboost_python3
clang: error: linker command failed with exit code 1 (use -v to see invocation)
error: command 'clang++' failed with exit status 1
world - [ERROR] launchables.world: Process Capture crashed with trace:
Traceback (most recent call last):
  File "/Users/Daniel/Projekty/PupilEyeTracker/pupil/pupil_src/launchables/world.py", line 113, in world
    import pupil_detectors
  File "/Users/Daniel/Projekty/PupilEyeTracker/pupil/pupil_src/shared_modules/pupil_detectors/__init__.py", line 16, in <module>
    build_cpp_extension()
  File "/Users/Daniel/Projekty/PupilEyeTracker/pupil/pupil_src/shared_modules/pupil_detectors/build.py", line 25, in build_cpp_extension
    ret = sp.check_output(build_cmd).decode(sys.stdout.encoding)
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 336, in check_output
    **kwargs).stdout
  File "/usr/local/Cellar/python3/3.6.4_2/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 418, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/usr/local/opt/python3/bin/python3.6', 'setup.py', 'install', '--install-lib=/Users/Daniel/Projekty/PupilEyeTracker/pupil/pupil_src/shared_modules']' returned non-zero exit status 1.

MainProcess - [INFO] os_utils: Re-enabled idle sleep.

Independently, I was able to reproduce a similar setup issue to yours after brew-upgrading `boost` and `python-boost`
```
world - [ERROR] launchables.world: Process Capture crashed with trace:
Traceback (most recent call last):
  File "/Users/pabloprietz/work/pupil/pupil_src/launchables/world.py", line 113, in world
    import pupil_detectors
  File "/Users/pabloprietz/work/pupil/pupil_src/shared_modules/pupil_detectors/__init__.py", line 18, in <module>
    from .detector_2d import Detector_2D
ImportError: dlopen(/Users/pabloprietz/work/pupil/pupil_src/shared_modules/pupil_detectors/detector_2d.cpython-36m-darwin.so, 2): Library not loaded: /usr/local/opt/boost-python/lib/libboost_python3.dylib
  Referenced from: /Users/pabloprietz/work/pupil/pupil_src/shared_modules/pupil_detectors/detector_2d.cpython-36m-darwin.so
  Reason: image not found
```

--- a/developer-docs/macos.md
+++ b/developer-docs/macos.md
@@ -56,7 +56,7 @@ brew tap homebrew/versions
 brew install glfw3
 # dependencies for 2d_3d c++ detector
 brew install boost
-brew install boost-python --with-python3
+brew install boost-python3
 brew install ceres-solver
 ```
 

